src/App.css

```diff
@@ -0,0 +1,42 @@
+.App {
+  text-align: center;
+  background-color: yellow;
+}
+
+.App__logo {
+  animation: App-logo-spin infinite 20s linear;
+  height: 40vmin;
+  pointer-events: none;
+}
+
+.App__header {
```

Comment:
I don't see any kss comments in here... Should there be?

Reply:
I don't think yet.  we are using Michaelangelo, a kss pre-formated helper, so I think that is what is currently doing the work.

Reply:
Ignore these App.css files, not sure how they got generated. I'm deleting them in the next commit here. You can find the kss styleguide annotations in App.scss. There's only a single one currently as a POC.

Reply:
As with my comment above, somehow the annotations got removed in the last commit. I've re-added them and it should make a bunch more sense here with the next commit.